webrender/src/device/gfx/device.rs

```diff
@@ -461,6 +493,16 @@ impl<B: hal::Backend> Device<B> {
             (limits.non_coherent_atom_size - 1) as usize,
         );
 
+        let quad_buffer = VertexBufferHandler::new(
+            &device,
+            &mut heaps,
+            hal::buffer::Usage::VERTEX,
+            &QUAD,
+            mem::size_of::<vertex_types::Vertex>(),
```

Comment:
why are we even providing the stride there if it can be derived from `T`?

Reply:
Removed this param in the last commit: https://github.com/szeged/webrender/pull/300/commits/b0579376553144ad5d712785d24cfbd8a8e0b533